src/test/java/jenkins/plugins/git/BrowsersJCasCCompatibilityTest.java

```diff
@@ -0,0 +1,175 @@
+package jenkins.plugins.git;
+
+import hudson.plugins.git.GitSCM;
+import hudson.plugins.git.browser.AssemblaWeb;
+import hudson.plugins.git.browser.BitbucketWeb;
+import hudson.plugins.git.browser.CGit;
+import hudson.plugins.git.browser.FisheyeGitRepositoryBrowser;
+import hudson.plugins.git.browser.GitBlitRepositoryBrowser;
+import hudson.plugins.git.browser.GitLab;
+import hudson.plugins.git.browser.GitList;
+import hudson.plugins.git.browser.GitRepositoryBrowser;
+import hudson.plugins.git.browser.GitWeb;
+import hudson.plugins.git.browser.GithubWeb;
+import hudson.plugins.git.browser.Gitiles;
+import hudson.plugins.git.browser.GitoriousWeb;
+import hudson.plugins.git.browser.GogsGit;
+import hudson.plugins.git.browser.KilnGit;
+import hudson.plugins.git.browser.Phabricator;
+import hudson.plugins.git.browser.RedmineWeb;
+import hudson.plugins.git.browser.RhodeCode;
+import hudson.plugins.git.browser.Stash;
+import hudson.plugins.git.browser.TFS2013GitRepositoryBrowser;
+import hudson.plugins.git.browser.ViewGitWeb;
+import hudson.scm.SCM;
+import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
+import org.jenkinsci.plugins.workflow.libs.GlobalLibraries;
+import org.jenkinsci.plugins.workflow.libs.LibraryConfiguration;
+import org.jenkinsci.plugins.workflow.libs.LibraryRetriever;
+import org.jenkinsci.plugins.workflow.libs.SCMRetriever;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.beans.HasPropertyWithValue.hasProperty;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.AllOf.allOf;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+public class BrowsersJCasCCompatibilityTest extends RoundTripAbstractTest {
+    @Override
+    protected void assertConfiguredAsExpected(RestartableJenkinsRule restartableJenkinsRule, String s) {
+        final List<LibraryConfiguration> libraries = GlobalLibraries.get().getLibraries();
+        assertThat(libraries, hasSize(19));
```

Comment:
This is an assertion that is very difficult to diagnose when it fails (at least for me).  I'd prefer that the assertion check a list of library names or another deeper attribute than the pure count of the libraries in the test.

Reply:
Happy to change it